iceoryx_hoofs/source/posix_wrapper/unix_domain_socket.cpp

```diff
@@ -128,11 +132,14 @@ UnixDomainSocket& UnixDomainSocket::operator=(UnixDomainSocket&& other) noexcept
 
         CreationPattern_t::operator=(std::move(other));
 
+        // NOLINTJUSTIFICATION will be fixed with refactoring in #1036
+        // NOLINTBEGIN(bugprone-use-after-move, hicpp-invalid-access-moved)
         m_name = std::move(other.m_name);
```

Comment:
Since the string owns the data anyway, this could just be a copy assignment which probably suppresses the whole warning.

There is no performance hit either, it already is a copy we are just lying here. Of course at some point `string` may support real move semantics, but I do not see how this would be possible while it owns the memory.

Reply:
I am not sure how the builder pattern will affect the issue of "fake" moves. It is a more fundamental design and paradigm issue IMO.

OK to postpone.